test/cmdlineTests/ir_optimized_transient_storage_value_type/args

```diff
@@ -0,0 +1 @@
+--via-ir --no-cbor-metadata --ir-optimized --debug-info none
```

Comment:
This one still needs to be updated to `--ir-optimized --optimize` (I left it like this since I'm currently investigating the `offsett_` issue, which doesn't appear with optimized output (i.e. no such helper function).